lib/ret_web/channels/hub_channel.ex

```diff
@@ -392,8 +547,68 @@ defmodule RetWeb.HubChannel do
   end
 
   def handle_out("naf" = event, payload, socket) do
+    sender_account = Account |> Ret.Repo.get(payload["sender_account_id"])
```

Comment:
It looks like this is going to do a database read on every NAF message, which isn't gonna work :(